public/setup/atomicupdate/updatepaths.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$sql[]="INSERT INTO `{$dbprefix}acces` (`nom`,`groupe_id`,`groupe`,`page`) VALUES ('Jours fériés','25','Gestion des jours fériés','/closingdays');";
```

Comment:
It's not an INSERT but an UPDATE we need. Please replace the entry with page = 'joursFeries/index.php'.

Also, you have to change the entry in menu table